### PR TITLE
GMode in Tourian Escape 3 and 4

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

File: region/tourian/main.json [hunk 1]
```diff
@@ -3970,6 +3970,40 @@
...
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "h_canArtificialMorphSpringBallBombJump"
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 5 }}
```
Comment (osse101) [inline region/tourian/main.json:3987]: With canHitBox, you can get through with 1PB on first pirate and 2 for the three pack.

File: region/tourian/main.json [hunk 2]
```diff
@@ -3970,6 +3970,55 @@
...
+                {
+                  "name": "G-Mode Morph PB Pirate Hitbox",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphSpringBallBombJump",
```
Comment (osse101) [inline region/tourian/main.json:4000]:
```
{"or": [
  "h_canArtificialMorphIBJ",
  "h_canArtificialMorphSpringBallBombJump"
]},
```

Comment (kjbranch) [reply region/tourian/main.json:4000]: if the pirate is still alive, the short ibj next to it is really annoying

Comment (osse101) [reply region/tourian/main.json:4000]: With bomb horizontally its fine.  Or just bomb jumping up extra high.

**Pull request merged**